README.md

```diff
@@ -3,56 +3,16 @@
 
 
 
-## Basic usage
+## Description
 
-To run an individual scenario, first generate a structure to hold the
-scenario parameters.  The scenario structure can be passed directly to
-the main model function.  
+`gcamland` replicates the land allocation decisions made in GCAM (https:://github.com/jgcri/gcam-core). The equations are documented in Wise et al. (2014). `gcamland` takes in prices and agricultural productivity growth and calculates land allocation. The model can operate in either hindcast (1975-2010) or future mode (2010-2100), running either single simulations or large ensembles.
 
-```R
-sceninfo <- ScenarioInfo(aScenarioType = "Hindcast")   # see helpfile for arguments 
-outdir <- run_model(sceninfo)
-```
 
-The results will be written as a series of csv files in the `outdir` directory.
-Subdirectories under that top-level directory will contain model outputs including
-price and yield expectations and land allocation.
+## Using `gcamland`
+A wiki describing how to use and modify `gcamland` is provided at: https://github.com/JGCRI/gcamland/wiki
 
+Please cite Wise et al. (2014) when using this package.
```

Comment:
You might want to add a `CITATION` file to the package too.

http://r-pkgs.had.co.nz/inst.html#inst-citation